rollup.config.cjs

```diff
@@ -34,7 +34,7 @@ module.exports = [
     ],
     output: {
       name: 'Chart',
-      file: 'dist/chart.js',
```

Comment:
this is a umd file. not a cjs file. we should serve umd to browsers and cjs to node

Reply:
@benmccann Fixed

Reply:
Now it looks backward compatible. Am I wrong?

Reply:
Adding `type: module` and adding `exports` are not backward compatible. But we can do this for v4